### ISSUE TITLE
Syntax error in main branch

### ISSUE THREAD

**Issue by AryanGodara** [Sudha247/ocaml-joy#47]
When I pulled the main branch, I noticed these two issues :-

## examples/dune
Current:
```
(executable
 (name rotate)
 (modules rotate)
 (libraries joy)))
 
(executable 
 (name line)
 (modules line)
 (libraries joy)
```
Should be
```
(executable
 (name rotate)
 (modules rotate)
 (libraries joy))
 
(executable 
 (name line)
 (modules line)
 (libraries joy))
```

## lib/shape.ml
Missing case for line
```
let rotate degrees shape = 
  match shape with 
  | Circle circle -> Circle { c = (rot circle.c degrees); radius = circle.radius }
  | Rectangle rectangle -> Rectangle { c = (rot rectangle.c degrees); length = rectangle.length; width = rectangle.width }
  | Ellipse ellipse -> Ellipse { c = (rot ellipse.c degrees); rx = ellipse.rx; ry = ellipse.ry }
```

Maybe something like:-
```
  | Line line -> Line {a = (rot line.a degrees); b = (rot line.b degrees)}
```

**Comment by joanita-51** [comment]
Thank you, @AryanGodara, for providing valuable insights into the issues

**Comment by nikochiko** [comment]
Thanks for debugging! @AryanGodara

**Comment by Sudha247** [comment]
Thanks @AryanGodara! We should fix this asap. Any takers for this? :)

**Comment by AryanGodara** [comment]
> Thanks @AryanGodara! We should fix this asap. Any takers for this? :)

Hi @Sudha247 , I believe @kushalpokharel has already implemented the fix in their new PR

**Comment by Sudha247** [comment]
Wonderful! I see this is being addressed in #34. Thanks @kushalpokharel!

**Comment by Sudha247** [comment]
I believe this is fixed by #34. Thanks @kushalpokharel for fixing it, and @AryanGodara for reporting.
